backend/services/mgmt/v1alpha1/connection-data-service/connection-data.go

```diff
@@ -370,11 +370,12 @@ func (s *Service) GetConnectionSchema(
 		schemas := []*mgmtv1alpha1.DatabaseColumn{}
 		for _, col := range dbschema {
 			schemas = append(schemas, &mgmtv1alpha1.DatabaseColumn{
-				Schema:     col.TableSchema,
-				Table:      col.TableName,
-				Column:     col.ColumnName,
-				DataType:   col.DataType,
-				IsNullable: col.IsNullable,
+				Schema:        col.TableSchema,
+				Table:         col.TableName,
+				Column:        col.ColumnName,
+				DataType:      col.DataType,
+				IsNullable:    col.IsNullable,
+				ColumnDefault: &col.ColumnDefault,
```

Comment:
Think we want to do the same thing as below where we only assign ColumnDefault if the value is not empty, that way it is truely nil.